app/org/sagebionetworks/bridge/models/accounts/User.java

```diff
@@ -30,16 +35,18 @@
     private String email;
     private String healthCode;
     private String studyKey;
-    private boolean consent;
-    private boolean signedMostRecentConsent;
     private SharingScope sharingScope;
     private Set<Roles> roles = Sets.newHashSet();
     private Set<String> dataGroups = Sets.newHashSet();
+    @JsonDeserialize(keyUsing = SubpopulationGuidDeserializer.class)
+    private Map<SubpopulationGuid,ConsentStatus> consentStatuses;
```

Comment:
As per IM conversation, we have a JSON serialization annotation on the User. Question #1, are we serializing User? If yes, we need to test it and make sure we aren't accidentally exposing health codes to end users. If not, we should remove the annotation.